kubernetes/linera-validator/build_and_redeploy.sh

```diff
@@ -90,7 +90,6 @@ else
         docker build \
             -f ../../docker/Dockerfile \
             ${copy:+--build-arg binaries=target/release} \
-            --build-arg environment=k8s-local \
```

Comment:
This is the default, so I removed it

Reply:
I think I added this because I preferred to be explicit about it — but I don't have strong feelings if you like it better this way :)

Reply:
I’m ok either way also! But if we prefer to be explicit (and enforce it), it might make sense to remove the defaults, and error if it’s not set